Adyen/UI/Form/FormItemViewBuilder.swift

```diff
@@ -59,4 +59,10 @@ public struct FormItemViewBuilder {
     public func build(with item: FormSeparatorItem) -> FormItemView<FormSeparatorItem> {
         FormSeparatorItemView(item: item)
     }
+
+    public static func renderItem(_ item: FormItem) -> AnyFormItemView {
```

Comment:
this name is misleading, its not actually rendering, its building, what's wrong with `build`?

Reply:
Good point 🤔